cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -64,16 +67,12 @@ describe('Script Builder', () => {
   const confirmSchemaComposition = () => {
     cy.getByTestID('flux-editor', {timeout: 30000})
       // we set a manual delay on page load, for composition initialization
-      // https://github.com/influxdata/ui/blob/e76f934c6af60e24c6356f4e4ce9b067e5a9d0d5/src/languageSupport/languages/flux/lsp/connection.ts#L435-L440
       .contains(`from(bucket: "${bucketName}")`, {timeout: 30000})
     cy.getByTestID('flux-editor').contains(
       `|> filter(fn: (r) => r._measurement == "${measurement}")`
     )
-    cy.getByTestID('flux-editor').contains(
-      `|> yield(name: "_editor_composition")`
-    )
     cy.getByTestID('flux-editor').within(() => {
-      cy.get('.composition-sync--on').should('have.length', 4) // four lines
+      cy.get('.composition-sync--on').should('have.length', 3) // three lines
```

Comment:
![](https://upload.wikimedia.org/wikipedia/en/2/29/Count_von_Count_kneeling.png)

>ha ha ha ha

(it's the little things you gotta do to keep yourself amused. I'll see myself out now)

Reply:
BRILLIANT!